ArcGISRuntimeSDKQt_CppSamples/Scenes/SyncMapViewSceneView/SyncMapViewSceneView.cpp

```diff
@@ -96,9 +95,7 @@ void SyncMapViewSceneView::setMapView(MapQuickView* mapView)
   {
     if (m_sceneView && m_mapView->isNavigating())
       m_sceneView->setViewpoint(m_mapView->currentViewpoint(ViewpointType::CenterAndScale), 0);
-
-  },
-  Qt::UniqueConnection); // lambda-unique-connection: Finds usages of Qt::UniqueConnection when the slot is a functor, lambda or non-member function. That connect() overload does not support Qt::UniqueConnection.
+  });
```

Comment:
This is just removing an unhelpful comment and an unsupported Qt::UniqueConnection?

Reply:
Good call. It was confusing to leave it that way. Nice cleanup.